cmd/thor/utils.go

```diff
@@ -67,7 +67,7 @@ type EthLogHandler struct {
 
 func initGethLogger(lvl ethlog.Lvl) {
 	handler := &EthLogHandler{
-		logger: log.New("pkg", "geth"),
+		logger: log.WithContext("pkg", "geth"),
```

Comment:
does it make sense to keep geth in thor ? maybe we can rename it to thor :D

Reply:
No this is actually used for internal geth logging. You can bump the log level and should get logs from the geth code that we use